### ISSUE TITLE
Store encryption keys locally by default

### ISSUE THREAD

**Issue by akontsevich** [vector-im/riot-web#9514]
Related to discussion in #9434.

For future: please add possibility (not default - only if user selects correspondent option in settings for certain PC/device - at least for on home for example) to store rooms encryption keys locally in settings unencrypted (or encrypted by account password) to prevent loosing message history in future. I think it is helpful to do at least for desktop version.

**Comment by aaronraimist** [comment]
Do you mean like automatically export it to a file? Because Riot has always had support for manually exporting keys to a file.

**Comment by akontsevich** [comment]
Yes, exactly. And if we create new encrypted rooms - automatically save keys for them as well.

**Comment by akontsevich** [comment]
For now I forced to store 3 passwords: login, local storage, server backup. And keep eye and every storage. Which is VERY inconvenient!!! What if I add new encrypted rooms: whether keys for them updated automatically in local storage or in server backup?! Think no, need to re-save them manually every time which is terrible and additional unnecessary headache!

Encryption should be convenient and transparent to users. Current functionality could scare away MANY MANY people from your platform and lead them to other where it works convenient (Wire, etc). Related to #9633.

**Comment by aaronraimist** [comment]
I'm not sure I understand. You are making more work for yourself. You don't have to export them locally if you have server side backup enabled. With server side backup enabled you don't ever have to touch it again. Your keys are automatically backed up.

Yes it would be nice to somehow automatically locally back it up as well, that is what you filed this issue for but you certainly don't have to manually do anything once you setup server side key backup.

**Comment by akontsevich** [comment]
@aaronraimist any way 2 passwords if server backup is ok to me. But what if do not like server side and want to do this locally? And there is another related problem with encryption, look at #9633.
